doc/release/1.15.0-notes.rst

```diff
@@ -122,6 +122,11 @@ Added experimental support for the 64-bit RISC-V architecture.
 Improvements
 ============
 
+``np.flip`` can operate over multiple axes
+------------------------------------------
+``np.flip`` now accepts tuples of int and None in its ``axis`` argument. If
```

Comment:
This is very easily misread as "tuples-of-in-and-None".  Perhaps "accepts None, or tuples of int, in its ... "